services/web/server/src/simcore_service_webserver/studies_dispatcher/_core.py

```diff
@@ -0,0 +1,101 @@
+import os
+import uuid
+from copy import deepcopy
+from dataclasses import dataclass
+from functools import lru_cache
+from typing import Iterator, Optional, Tuple
+
+from aiohttp import web
+from pydantic import ValidationError
+
+MEGABYTES = 1024*2014
+
+
+# VIEWERS MAP -----------------------------------------------------------------------------
+class MatchNotFoundError(Exception):
+    def __init__(self, reason):
+        super().__init__()
+        self.reason = reason
+
+
+@dataclass
+class ViewerInfo:
+    key: str
+    version: str
+    label: str
+    input_port_key: str = (
+        "input_1"  # name of the connection port, since it is service-dependent
+    )
```

Comment:
May I suggest it like so?
```suggestion
    # name of the connection port, since it is service-dependent
    input_port_key: str = "input_1"
```